AppGenGuide.md

```diff
@@ -30,139 +33,121 @@ application to be deployed as a StarlingX App it needs to be designed so it can
-    - App Metadata Configuration
-    - App Setup Configuration
+- [App manifest configuration](#app-manifest-configuration)
+- [Metadata file configuration](#metadata-file-configuration)
+- [App Setup Configuration](#app-setup-configuration)
 
 ### App manifest configuration
 
-5. In this stage the fields inside the **appManifestFile-config** will be configured
-
-   ![app manifest config](BuildGuide/app-manifest-config-empty.png)
-
-   These are the minimum required fields that will need to be filled in order
-   for the app-generator to work properly.
-
-   Brief fields explanation:
-   - **appName**: desired StarlingX application name
-   - **appVersion**: desired StarlingX application version
-   - **namespace**: desired StarlingX application namespace (note that this
-     namespace is not the same as the Kubernetes namespace)
-   - **chart**: create a copy of name, version, path and chartGroup for every
-     helm-chart in your application
-     - **name**: your helm-chart name as it is in the chart metadata
-     - **version**: your app version as it is in the chart metadata
-     - **path**: helm-chart directory, helm repo or helm tgz path. Currently
-       only helm-chart in directories have been tested
-     - **chartGroup**: default is _application-name-charts_
-   - **chartGroup**
-     - **name**: Only one chart group per application
-     - **chart_names**: a list of the names of the charts from your application
+In this stage the section **appManifestFile-config** from the
+`app_manifest.yaml` will be configured.
+
+![app manifest config](BuildGuide/app-manifest-config-empty.png)
+
+These are the minimum required fields that will need to be filled in order
+for the StarlingX App Generator to work properly.
+
+Below you will find a brief explanation of every one of the required fields
+which will help you fill them out for you application:
+
+- **appName** field: desired StarlingX application name, referenced throughout
+the whole system.
+- **appVersion** field: the version of the application that the generated
+package will represent.
+- **namespace** field: desired StarlingX application namespace (note that this
+namespace is not the same as the Kubernetes namespace).
+- **chart** section: an array with an object for every Helm chart in your
+application. Each object contains:
+  - **name** field: your Helm chart name as it is in the chart metadata.
+  - **version** field: your app version as it is in the chart metadata.
+  - **path** field: relative path to the Helm chart directory, Helm repo or
+  Helm package file. 
+  > _NOTE_: Currently only Helm charts in directories have been tested.
+  - **chartGroup** field: default is _application-name-charts_.
+- **chartGroup** section:
+  - **name**: only one chart group per application.
+  - **chart_names**: a list of the names of the charts from your application.
+
+> _NOTE_: TODO can't this be automated since it's just a copy from other values already set?
```

Comment:
It can if we choose the default chartGroup name to be used.

Reply:
Yeah, we can. It can be the default, or just a value inside the yaml. I did think of getting rid of this section and create the variable inside the code (this chartgroup section exists in order to create a variable containing all helm-chart names), but didn't act on it

Reply:
@Danmcaires @tnpb-encora and I added a TODO in the `README.md` to address this.